### PR TITLE
again some changes adding Live and work on cropping

### PR DESCRIPTION
**Pull request by ModusMorris**
I have already worked a bit on the live and improved the cropping again, now it also creates a folder for the users after cropping and puts the pictures in there. I haven't been able to test live yet, but we need the network as soon as it's up and running so we can start testing and see how good our network is. But so far everything should work now. in the train.py I made two changes so that you can also go over the Batch_Size 8, can sometimes be helpful.

### CODE REVIEW

File: network.py
Comment (heyitsalina) [inline network.py:1]: i looked into pepes pull request a few days ago where he finished the network and balanced accuracy files 
it looks like you have gotten the wrong, old version of the network in one line 
line 26 
`nn.Flatten(), nn.Linear(524288, nClasses))` 
was deleted which was what pepe improved, while the old incorrect line was added (perhaps a merging/pulling mistake) 

so maybe edit that part back to the deleted line and i can merge :))
the rest looks fine to me, thank you for the work!

**Review by heyitsalina — APPROVED** [review]
merging for now, thanks maurice!
